.graphqlconfig.yml

```diff
@@ -13,3 +13,6 @@ projects:
         region: us-east-1
         apiId: null
         maxDepth: 2
+extensions:
```

Comment:
I don't know if this is needed? this happened when I did amplify push I think

Reply:
👍